tests/Doctrine/Tests/Models/GH5804/GH5804Article.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Doctrine\Tests\Models\GH5804;
+
+/**
+ * @Entity
+ * @Table(name="gh5804_articles")
+ */
+class GH5804Article
+{
+    /**
+     * @Id
+     * @Column(type="GH5804Type")
+     * @GeneratedValue(strategy="CUSTOM")
+     * @CustomIdGenerator(class="\Doctrine\Tests\Models\GH5804\GH5804Generator")
```

Comment:
I guessed it was that way, but my local tests fail. I even update merged with the latest `master` release, updated the dependencies (my Annotations version was right, anyway), but your suggestion (which seems to be logical) doesn't work to me. Did you try by yourself?
Anyway, even without that `::class` notation, everything is fine.